test/images/Makefile

```diff
@@ -0,0 +1,51 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+REGISTRY ?= gcr.io/google_containers
+GOARM=7
+GOLANG_VERSION=1.7.5
+QEMUVERSION=v2.7.0
+export
+
+ifndef WHAT
+$(error WHAT is a required variable, ex: make all WHAT=net)
+endif
+
+# Build code.
+#
+# Args:
+#   WHAT: Directory names to build.
+#
+# Example:
+#   make all WHAT=clusterapi-tester
+all: all-container
+
+sub-container-%:
+	$(MAKE) build-image TEST=$*
+
+sub-push-%:
+	$(MAKE) push-image TEST=$*
+
+all-container: $(addprefix sub-container-,$(WHAT))
```

Comment:
Initially I had a logic to build all the images in this folder, based on review comment from @luxas I changed logic to make WHAT as compulsory argument,

more infor - https://github.com/kubernetes/kubernetes/pull/44910#discussion_r121389369